lib/taskana-core/src/main/java/pro/taskana/common/internal/jobs/AbstractTaskanaJob.java

```diff
@@ -13,8 +13,8 @@
 /** Abstract base for all background jobs of TASKANA. */
 public abstract class AbstractTaskanaJob implements TaskanaJob {
 
-  protected final Instant firstRun;
-  protected final Duration runEvery;
+  protected Instant firstRun;
+  protected Duration runEvery;
```

Comment:
Why has the final been removed?

Reply:
TaskUpdate Job has his own firstRun and runEvery config and needs to overwrite the abstract class ... looks like technical debt that should be refactored.